client/src/components/DataDisplay/TheTrafficPerPage.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <v-card>
+        <v-card-title>Traffic per Page</v-card-title>
+
+        <v-card-text>
+            <v-row>
+                <v-col cols="6">
+                    <base-traffic-card title="Homepage" :traffic="homepageTraffic" />
+                </v-col>
+
+                <v-col cols="6">
+                    <base-traffic-card title="Product 1 Page" :traffic="product1pageTraffic" />
+                </v-col>
+            </v-row>
+
+            <v-row>
+                <v-col cols="6">
+                    <base-traffic-card title="Product 2 Page" :traffic="product2pageTraffic" />
+                </v-col>
+
+                <v-col cols="6">
+                    <base-traffic-card title="Product 3 Page" :traffic="product3pageTraffic" />
+                </v-col>
+            </v-row>
+        </v-card-text>
+    </v-card>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+export default {
+    components: {
+        baseTrafficCard: () => import('@/components/UI/BaseTrafficCard')
+    },
+
+    computed: {
```

Comment:
here you can build an array with specific items, then make v-for in template and one method with one argument instead of several computed

Reply:
@daniel-pupczyk-rmtcfm-com Ok. I'm gonna try.

Reply:
you can leave it for now - we will refactor code after backend, we don't have a lot of time